lib/src/usage.dart

```diff
@@ -171,8 +176,52 @@ class Usage {
     numHelpLines = 0;
   }
 
+  /// Wrap a single line of text into lines no longer than length.
+  /// Try to split at whitespace, but if that's not good enough to keep it
+  /// under the length, then split in the middle of a word.
+  List<String> wrap(String text, int length) {
```

Comment:
This is only called by `write()`, right?

How about:

* Make it private.
* Move `const int minColumnWidth = 10;` and the width calculation into here instead of passing it in.

Reply:
Good suggestions. Done!